frontend/src/HomePage/SignHandlers/SignUp.jsx

```diff
@@ -0,0 +1,50 @@
+import Topbar from '../Top/Topbar';
+import { submitProfile } from '../../../utils';
+import './SignUp.css'
+
+function SignUp() {
+    /* Function to handle the immedate submision of profile creation form
+    extracts from the inputs and updates the state to track data
+    accordingly and passes this info onto the client-side endpoint helper [submitProfile]
+    */
+    const handleSignUp = async (e) => {
+        e.preventDefault();
+        const form = e.target;
+        const userData = {
+            email: form.elements.email.value,
+            password: form.elements.password.value
+        };
+        try {
+            await submitProfile(userData);
```

Comment:
It may be a good idea to show a loading state - maybe something to do in a future PR